ruma-events/src/room/message.rs

```diff
@@ -585,3 +615,15 @@ pub struct KeyVerificationRequestEventContent {
     /// events that have a m.reference relationship with this event.
     pub to: UserId,
 }
+
+/// The payload for a custom message event.
+#[doc(hidden)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct CustomEventContent {
+    /// A custom msgtype
+    pub msgtype: String,
```

Comment:
No need for the fields in this type to be `pub`lic.

Reply:
These are public for the time being (for tests), Ill make these private as soon as the public API is ready.